hi-ml-cpath/src/health_cpath/preprocessing/create_tiles_dataset.py

```diff
@@ -109,7 +109,9 @@ def get_tile_info(sample: Dict[SlideKey, Any], occupancy: float, tile_box: Box,
         TileKey.TILE_RIGHT.value: tile_box.x + tile_box.w,
         TileKey.TILE_BOTTOM.value: tile_box.y + tile_box.h,
         TileKey.OCCUPANCY: occupancy,
-        TileKey.SLIDE_METADATA: {TileKey.from_slide_metadata_key(key): value
+        # TileKey.SLIDE_METADATA: {TileKey.from_slide_metadata_key(key): value
+        #                          for key, value in sample[SlideKey.METADATA].items()
```

Comment:
You may wish to clean up this commented-out fragment (and below), or add a descriptive comment in case it was left deliberately for reference.